.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: ['3.8']
-        toxenv: [quality, django32]
+        toxenv: [quality, django32, django42]
         exclude:
```

Comment:
Let's remove this exclude section so quality checks also start to run.

Reply:
ok